tests/model_hub_tests/pytorch/models/nightly

```diff
@@ -0,0 +1,34 @@
+hf-tiny-model-private/tiny-random-BartForCausalLM
```

Comment:
may be it is better to name as `hf_llms_nightly`?

Reply:
or `tiny-random-models-precommit`

Reply:
Ok, let's combine everything we've got so far. The names would be _hf-tiny-random-models-nightly_ and _hf-tiny-random-models-precommit_.